crates/dojo-physics/examples/projectile/src/projectile.cairo

```diff
@@ -0,0 +1,245 @@
+use core::option::OptionTrait;
+use array::ArrayTrait;
+use debug::PrintTrait;
+use array::ArrayTCloneImpl;
+use array::SpanTrait;
+use clone::Clone;
+use traits::PartialOrd;
+
+use cubit::test::helpers::assert_precise;
+use cubit::types::fixed::{Fixed, FixedPartialOrd, FixedTrait, ONE_u128};
+use cubit::math::trig;
+
+use dojo_physics::vec2::{Vec2, Vec2Trait};
+
+fn main() -> (usize, Array::<Fixed>, Array::<Fixed>) {
```

Comment:
Once we extract the helpers, perhaps we can turn this into a test file for `dojo-physics` rather than an example?

Reply:
that sounds great. I think I can get this done soon ish